docs/hsi.xml

```diff
@@ -0,0 +1,875 @@
+      </para>
+      <itemizedlist>
+        <listitem>
+          <para>
+            For HSI-3 this should be set <emphasis>v1.5.0</emphasis>
+          </para>
+        </listitem>
+      </itemizedlist>
+    </refsect3>
+
+    <refsect1 id="conclusions">
+      <title>Conclusion</title>
+      <para>
+        Any system with a Host Security ID of <code>0</code> can easily be
+        modified from userspace.
+        PCs with confidential documents should have a <code>HSI:3</code> or
+        higher level of protection.
+        In a graphical tool that would show details about the computer (such as
+        GNOME Control Center’s details tab) the OS could display a field
+        indicating Host Security ID.
+        The ID should be shown with an alert color if the security is not at
+        least <code>HSI:1</code> or the suffix is <code>!</code>.
+      </para>
+      <para>
+        On Linux <code>fwupd</code> is used to enumerate and update firmware.
+        It exports a property <code>HostSecurityId</code> and a
+        <code>GetHostSecurityAttrs()</code> method.
+        The attributes are supposed to represent the <emphasis>system as a whole</emphasis>
+        but individual (internal) devices are able to make a claim that they
+        worsened the state of the security of the system.
+        Certain attributes can “obsolete” other attributes.
+        An example is BIOSGuard will set obsoletes to <code>org.intel.prx</code>.
+      </para>
+      <para>
+        A plugin method gets called on each plugin which adds attributes directly
+        from the hardware or kernel.
+        Several attributes may be dependent upon the kernel performing measurements
+        and it will take time for these to be upstreamed.
+        In some cases security level measurements will only be possible on systems
+        with a newer kernel.
+      </para>
+      <para>
+        The long term goal is to increase the <code>HSI:x</code> level of systems
+        being sold to consumers.
+        By making some of the <code>HSI:x</code> attributes part of the LVFS
+        uploaded report we can allow users to compare vendors and models before
+        purchasing hardware.
+      </para>
+    </refsect1>
+
+    <refsect2 id="ommissions">
+      <title>Intentional Omissions</title>
+    </refsect2>
+    <refsect3 id="intel-txt">
+      <title>Intel TXT</title>
+      <para>
+        This security functionality requires a binary blob to operate.
+        It is not widely used and its presence is not tested.
+      </para>
+    </refsect3>
+    <refsect3 id="intel-sgx">
+      <title>Intel SGX</title>
+      <para>
+        This is not widely used as it has several high severity security issues.
+      </para>
+    </refsect3>
+    <refsect3 id="intel-mpx">
+      <title>Intel MPX</title>
+      <para>
+        MPX support has now been removed from popular compilers and the kernel
+        several years ago and it is now considered obsolete.
```

Comment:
this statement won't age well - probably need a specific time frame to include

Reply:
How about

        MPX support was removed from GCC and the Linux kernel in 2019 and it is
        now considered obsolete.

Reply:
👍